lib/siwapp/schema/series.ex

```diff
@@ -11,4 +12,12 @@ defmodule Siwapp.Schema.Series do
     field :first_number, :integer, default: 1
     has_many :invoices, Invoice
   end
+
+  def changeset(series, attrs) do
+    keys = Map.keys(attrs)
```

Comment:
this is a bad idea, because you'll cast any attribute which is inside attrs, which could be anything.